service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/PartyAddedToCaseEventHandler.java

```diff
@@ -8,47 +8,60 @@
 import uk.gov.hmcts.reform.fpl.service.representative.RepresentativeNotificationService;
 
 import java.util.List;
 
 import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE_CHILD_NAME;
 import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_THROUGH_DIGITAL_SERVICE_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_THROUGH_DIGITAL_SERVICE_NOTIFICATION_TEMPLATE_WITH_CHILD;
 import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.DIGITAL_SERVICE;
 import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.EMAIL;
 
 @Service
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class PartyAddedToCaseEventHandler {
-    private final PartyAddedToCaseContentProvider partyAddedToCaseContentProvider;
+    private final PartyAddedToCaseContentProvider contentProvider;
     private final RepresentativeNotificationService representativeNotificationService;
     private final RepresentativeService representativeService;
+    private final FeatureToggleService toggleService;
 
     @EventListener
     public void notifyParties(final PartyAddedToCaseEvent event) {
         CaseData caseData = event.getCaseData();
         CaseData caseDataBefore = event.getCaseDataBefore();
 
         List<Representative> representativesServedByDigitalService = representativeService.getUpdatedRepresentatives(
-            caseData.getRepresentatives(), caseDataBefore.getRepresentatives(), DIGITAL_SERVICE);
+            caseData.getRepresentatives(), caseDataBefore.getRepresentatives(), DIGITAL_SERVICE
+        );
         List<Representative> representativesServedByEmail = representativeService.getUpdatedRepresentatives(
-            caseData.getRepresentatives(), caseDataBefore.getRepresentatives(), EMAIL);
+            caseData.getRepresentatives(), caseDataBefore.getRepresentatives(), EMAIL
+        );
 
-        NotifyData servedByEmailParameters = partyAddedToCaseContentProvider
-            .getPartyAddedToCaseNotificationParameters(caseData, EMAIL);
+        String template = toggleService.isEldestChildLastNameEnabled()
+                          ? PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE_CHILD_NAME
+                          : PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE;
+
+        NotifyData servedByEmailParameters = contentProvider.getPartyAddedToCaseNotificationParameters(caseData, EMAIL);
 
         representativeNotificationService.sendToUpdatedRepresentatives(
-            PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE,
-            servedByEmailParameters, caseData, representativesServedByEmail);
+            template, servedByEmailParameters, caseData, representativesServedByEmail
+        );
+
+        template = toggleService.isEldestChildLastNameEnabled()
```

Comment:
I'd use a different local variable instead of reusing template

Reply:
done